DQM/SiTrackerPhase2/python/Phase2TrackerMonitorDigi_cfi.py

```diff
@@ -11,88 +11,124 @@
+           Nybins = cms.int32(32),
+           ymin   = cms.double(0.5),
+           ymax   = cms.double(32.5),
+           switch = cms.bool(True)
     ),
     EtaH = cms.PSet(
         Nbins  = cms.int32(45),
         xmin   = cms.double(-4.5),
-        xmax   = cms.double(4.5)
+        xmax   = cms.double(4.5),
+        switch = cms.bool(True)
     ),
     DigiChargeH = cms.PSet(
       Nbins = cms.int32(261),
       xmin   = cms.double(0.5),
-      xmax   = cms.double(260.5)
+      xmax   = cms.double(260.5),
+        switch = cms.bool(True)
     ), 
     TotalNumberOfDigisPerLayerH = cms.PSet(
       Nbins = cms.int32(100),
       xmin   = cms.double(0.0),
-      xmax   = cms.double(50000.0)
+      xmax   = cms.double(50000.0),
+        switch = cms.bool(True)
     ),
     NumberOfHitDetsPerLayerH = cms.PSet(
       Nbins = cms.int32(2000),
       xmin   = cms.double(-0.5),
-      xmax   = cms.double(2000.5)
+      xmax   = cms.double(2000.5),
+        switch = cms.bool(True)
     ),
     NumberOfClustersPerDetH = cms.PSet(
            Nbins = cms.int32(100),
            xmin = cms.double(-0.5),
-           xmax = cms.double(99.5)
+           xmax = cms.double(99.5),
+           switch = cms.bool(True)
     ),
     ClusterWidthH = cms.PSet(
            Nbins = cms.int32(16),
            xmin   = cms.double(-0.5),
            xmax   = cms.double(15.5),
+           switch = cms.bool(True)
     ),
     ClusterChargeH = cms.PSet(
         Nbins = cms.int32(1024),
         xmin   = cms.double(0.5),
-        xmax   = cms.double(1024.5)
+        xmax   = cms.double(1024.5),
+        switch = cms.bool(True)
     ),  
-    ClusterPositionH = cms.PSet(
-        Nbins = cms.int32(1016),
+    ClusterPositionSH = cms.PSet(
+        Nxbins = cms.int32(1016),
         xmin   = cms.double(0.5),
-        xmax   = cms.double(1016.5)
-    ),  
+        xmax   = cms.double(1016.5),
+        Nybins = cms.int32(2),
+        ymin   = cms.double(0.5),
+        ymax   = cms.double(2.5),
+        switch = cms.bool(True)
+    ),
+    ClusterPositionPH = cms.PSet(
+        Nxbins = cms.int32(1016),
+        xmin   = cms.double(0.5),
+        xmax   = cms.double(1016.5),
+        Nybins = cms.int32(32),
+        ymin   = cms.double(0.5),
+        ymax   = cms.double(32.5),
+        switch = cms.bool(True)
+    ),
     XYPositionMapH = cms.PSet(
         Nxbins = cms.int32(1250),
         xmin   = cms.double(-1250.),
         xmax   = cms.double(1250.),
         Nybins = cms.int32(1250),
         ymin   = cms.double(-1250.),
-        ymax   = cms.double(1250.)
+        ymax   = cms.double(1250.),
+        switch = cms.bool(False)
     ),
     RZPositionMapH = cms.PSet(
         Nxbins = cms.int32(3000),
         xmin   = cms.double(-3000.),
         xmax   = cms.double(3000.),
         Nybins = cms.int32(1250),
         ymin   = cms.double(0.),
-        ymax   = cms.double(1250.)
+        ymax   = cms.double(1250.),
+        switch = cms.bool(False)
     )
 )
```

Comment:
@gsaha009.
I think that in this file `InnerPixelDigiSource` should be customized in the case of premixing.
Something like:
```
from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
(premix_stage2 & phase2_tracker).toModify(digiMon,
    InnerPixelDigiSource = "mixData:Pixel"
)
```
or something similar (see e.g. https://cmssdt.cern.ch/dxr/CMSSW/source/RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizer_cfi.py#31)